airbyte-webapp/src/components/connectorBuilder/types.ts

```diff
@@ -99,7 +99,7 @@ export interface BuilderStream {
 
 export const DEFAULT_BUILDER_FORM_VALUES: BuilderFormValues = {
   global: {
-    connectorName: "",
+    connectorName: "Untitled",
```

Comment:
We want this eventually anyway, and this prevents us from showing a `Required` error on Reset All